letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +155,37 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    def test_simple_verify(self, mock_http01_verify, mock_tlssni01_verify):
+        # pylint: disable=protected-access
+        key = jose.JWK.load(test_util.load_vector('rsa512_key.pem'))
```

Comment:
Oh, I see.

I'll read a bit about how mock.patch works and then add another comment, or replace the key with a dummy one.

Thanks for pointing it out! :)